deploy/effective-dart-rules/bin/main.dart

```diff
@@ -9,53 +8,45 @@ import 'package:path/path.dart' as path;
 final _unescape = HtmlUnescape();
 final _anchorPattern = RegExp(r'(.+)\{#([^#]+)\}');
 
-Future<Null> main(List<String> arguments) async {
-  var dirPath = "src/_guides/language/effective-dart";
-  var filenames = const [
-    "style.md",
-    "documentation.md",
-    "usage.md",
-    "design.md"
-  ];
+Future<void> main(List<String> arguments) async {
+  const dirPath = 'src/_guides/language/effective-dart';
+  const fileNames = ['style.md', 'documentation.md', 'usage.md', 'design.md'];
 
-  var sections = filenames.map((name) => Section(dirPath, name)).toList();
+  var sections = fileNames.map((name) => Section(dirPath, name)).toList();
```

Comment:
I reacted to this change. Turns out, I consider `filename` to be an actual word, not a compound phrase. 

See https://english.stackexchange.com/questions/5366/which-is-correct-filename-file-name-or-filename for some fun commentary.

Reply:
Good catch! We use one word: https://developers.google.com/style/word-list#filename

Reply:
I forgot about this usage, thanks for catching this :)